examples/advanced_examples/plot_mne_features_pipeline.py

```diff
@@ -0,0 +1,97 @@
+"""
+==========================
+Pipelines using the mne-features library
+==========================
+
+This example shows how to evaluate a pipeline constructed using the
+mne-features library
+
```

Comment:
What is mne-features, and what is computing?

Reply:
Some introduction is needed here...

We expected some basic math, like this: https://roche.github.io/neuro-meeglet/background/1_background_wavelets.html

and 

with the same amount of details of this:

https://roche.github.io/neuro-meeglet/tutorials/plot_eeg_covariance.html